ui/page/settingsCreator/view.jsx

```diff
@@ -439,4 +464,5 @@ const HELP = {
   BLOCKED_WORDS: 'Comments and livestream chat containing these words will be blocked.',
   MODERATORS: 'Moderators can block channels on your behalf. Blocked channels will appear in your "Blocked and Muted" list.',
   MODERATOR_SEARCH: 'Enter a channel name or URL to add as a moderator.\nExamples:\n - @channel\n - @channel#3\n - https://odysee.com/@Odysee:8\n - lbry://@Odysee#8',
+  MEMBERS_ONLY_CHAT: 'Only channel members with "Members-only chat" perk can participate in public livestream chats',
```

Comment:
:small_red_triangle: Punctuate please. There's already lots of legacy ones, and we're tired of cleaning up new ones.

If this is a typo, then fine. But considering the repeated behavior and sometimes intentional removal, it's probably not.

I don't want to start another style debate, but for now we punctuate for subtitles and almost everything.  Only titles and button labels aren't.